File: go/pangram/pangram.go
package pangram

import "strings"

func IsPangram(s string) bool {
	s = strings.ToLower(s)

	for c := 'a'; c <= 'z'; c++ {
		if strings.IndexRune(s, c) == -1 {
			return false
		}
	}
	return true
}

File: cpp/pangram/pangram.cpp
#include "pangram.h"

#include <unordered_set>

using namespace std;
namespace pangram {

    bool is_pangram(const string &sentence) {
        unordered_set<char> charsInSentence;
        for (const auto c: sentence)
            if (isalpha(c)) charsInSentence.emplace(tolower(c));

        return charsInSentence.size() == 26;

    }
}
